client/app/app.js

```diff
@@ -1,79 +1,133 @@
+    "ovh-angular-responsive-page-switcher",
+    "ovh-angular-responsive-popover",
+    "ovh-angular-sidebar-menu",
 
-  "ng-slide-down",
-  "ovh-angular-jsplumb",
-  "tmh.dynamicLocale",
-  "ovh-angular-module-status",
-  "ovh-angular-otrs",
-  "ovh-api-services",
+    "ng-slide-down",
+    "ovh-angular-jsplumb",
+    "tmh.dynamicLocale",
+    "ovh-angular-module-status",
+    "ovh-angular-otrs",
+    "ovh-api-services",
 
-  "ovh-jquery-ui-draggable-ng",
-  "ovh-angular-jquery-ui-droppable",
-  "ovh-angular-slider",
-  "ng-at-internet",
-  "atInternetUiRouterPlugin",
-  "ngFlash",
-  "matchmedia-ng",
-  "ovh-angular-user-pref",
-  "ovh-angular-doc-url",
-  "ovhBrowserAlert"
+    "ovh-jquery-ui-draggable-ng",
+    "ovh-angular-jquery-ui-droppable",
+    "ovh-angular-slider",
+    "ng-at-internet",
+    "atInternetUiRouterPlugin",
+    "ngFlash",
+    "matchmedia-ng",
+    "ovh-angular-user-pref",
+    "ovh-angular-doc-url",
+    "ovhBrowserAlert"
 ])
+    .config(function ($stateProvider, TranslateDecoratorServiceProvider, TranslateServiceProvider) {
+        "use strict";
 
-.config(function ($stateProvider, TranslateDecoratorServiceProvider, TranslateServiceProvider) {
-    "use strict";
+        // Config current locale
+        TranslateServiceProvider.setUserLocale();
 
-    // Config current locale
-    TranslateServiceProvider.setUserLocale();
+        // Add translations decorator (need to be added before routes definitions)
+        TranslateDecoratorServiceProvider.add($stateProvider);
+    })
+    .config(function ($urlRouterProvider, $locationProvider) {
+        "use strict";
+        $urlRouterProvider.otherwise("/");
+        $locationProvider.html5Mode(false);
+    })
+    .config(function (responsivePopoverProvider) {
+        // tell to the module that we consider a mobile device with at least 800px width
+        responsivePopoverProvider.setMobileMediaQuery("(max-width: 800px)");
+    })
+    .config(ChartJsProvider => {
+        ChartJsProvider.setOptions({ colors: ["#3DD1F0", "#00ADF9", "#DCDCDC", "#46BFBD", "#FDB45C", "#949FB1", "#4D5360"] });
+    })
+    .config(ouiTableConfigurationProvider => {
+        ouiTableConfigurationProvider.setCssConfig({
+            tablePanel: "oui-table-panel",
+            table: "oui-table oui-table_responsive",
+            thead: "oui-table__headers",
+            tbody: "oui-table__body",
+            tr: "oui-table__row cui-dropdown-menu-container",
+            th: "oui-table__header",
+            td: "oui-table__cell",
+            sortable: "oui-table__cell_sortable oui-table__cell_sortable-asc",
+            sorted: "oui-table__cell_sorted",
+            sortableAsc: "oui-table__cell_sortable-asc",
+            sortableDesc: "oui-table__cell_sortable-desc",
+            closed: "oui-table__row_closed",
+            emptyTable: "oui-table-empty"
+        })
+            .setPageSize(10)
+            .setExpandButtonTemplate(`
+                <i role="button"
```

Comment:
not sure about this indent

Reply:
eslint expect an indent after a function call if chaining function calls.  The same apply for promises.

bla()
.then() => error.

bla()
    .then() => ok.